content/en/internal/configuration-database/service-types/_index.md

```diff
@@ -0,0 +1,109 @@
+- [site.SAM](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=site.SAM): (Regional Service) Site-level SAM monitoring box
+- [ngi.OpsPortal](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=ngi.OpsPortal): (Regional service) NGI-level regional operations portal instance
+- [eu.egi.MPI](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=eu.egi.MPI): Defines a dummy Service Type to enable the running of MPI tests for services providing MPI capabilities. Sites must have one instance of this Service Type associated with a CREAM-CE service.
+- [argo.poem](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=argo.poem): POEM is system for managing profiles of probes and metrics in ARGO system.
+- [argo.mon](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=argo.mon): ARGO Monitoring Engine gathers monitoring metrics and publishes to messaging service.
+- [argo.consumer](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=argo.consumer): ARGO Consumer collects monitoring metrics from monitoring engines.
+- [argo.computeengine](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=argo.computeengine): ARGO Compute Engine computes availability and reliability of services.
+- [argo.api](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=argo.api): ARGO API service for retrieving status and A/R results.
+- [argo.webui](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=argo.webui): ARGO web user interface for metric A/R visualization and recalculation management.
+- [egi.aai.saml](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=egi.aai.saml): EGI AAI CheckIn SAML interface. Enables federated access to EGI services and resources using Security Assertion Markup Language (SAML). Provided by GRNET.
+- [egi.aai.oidc](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=egi.aai.oidc): EGI AAI CheckIn OpenID Connect interface. Enables federated access to EGI services and resources using OpenID Connect (OIDC). Provided by GRNET.
+- [egi.aai.tts](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&service_type=egi.aai.tts): EGI AAI CheckIn token translation service. Enables the translation between different authentication and authorisation protocols. Provided by GRNET.
+
+<!-- markdownlint-enable no-inline-html -->
+     
+## Adding new services types
+
+Please feel free to make a request for a new service type. For CUSTOM service
+types, we would like to make this process as light-weight as possible. However,
+currently all new service type requests need to be assessed by EGI via
+lightweight review process (by OMB and EGI Ops) so that only suitable types are
+added to GOCDB and to prevent duplication.
```

Comment:
```suggestion
Please feel free to make a request for a new service type. All new service type requests need
to be assessed by EGI via lightweight review process (by OMB and EGI Ops) so that only
suitable types are added to GOCDB and to prevent duplication.
```

It looks like this page has dropped the "Custom Service Types" section, I agree with this - but lets drop this refernece to CUSTOM service types.